extensions/interactions/EndExploration/directives/end-exploration-validation.service.ts

```diff
@@ -61,6 +61,19 @@ export class EndExplorationValidationService {
         });
       }
     }
+    // Validate invalid exploration ids given in author recommended
+    // explorations ids.
+    let warningMessageElement = document.querySelector(
+      'oppia-interactive-end-exploration .oppia-error-message-text span'
+    );
+    if (warningMessageElement) {
+      let warningMessage = warningMessageElement.textContent;
+      warningMessage &&
+      warningsList.push({
+        type: AppConstants.WARNING_TYPES.ERROR,
+        message: warningMessage
+      });
```

Comment:
Lets rather use `if (warningMessage) {` this is conceptually the same but from readability perspective I think `if` is better.

Reply:
Done!